lib/Jobs/PasswordExpirationNotifierJob.php

```diff
@@ -147,7 +147,7 @@ private function sendAboutToExpireNotification(OldPassword $passInfo, $expiratio
 		$notification->setApp('password_policy')
 			->setUser($passInfo->getUid())
 			->setDateTime(new \DateTime("@{$notificationTimestamp}"))
-			->setObject('about_to_expire', $passInfo->getId())
+			->setObject('about_to_expire', (string)$passInfo->getId())
```

Comment:
Please share the error which is fixed through this

Reply:
``$passInfo->getId()`` returns an ``int`` but ``setObject()`` expects a string.